src/types/grants.ts

```diff
@@ -402,6 +425,12 @@ export type TeamMember = {
   relevantLink?: string
 }
 
+export type Milestone = {
```

Comment:
For projects we have this

```typescript
export type ProjectMilestone = {
  id: string
  project_id: string
  title: string
  description: string
  status: MilestoneStatus
  updated_by?: string
  updated_at?: Date
  created_by: string
  created_at: Date
}

```

And I'm noticing we'd be missing the estimated completion date.

Maybe it should be 

```typescript
export type ProjectMilestone = Milestone & {
  id: string
  project_id: string
  status: MilestoneStatus
  updated_by?: string
  updated_at?: Date
  created_by: string
  created_at: Date
}
```